pkg/domain/infra/abi/trust.go

```diff
@@ -142,15 +142,15 @@ func getPolicyShowOutput(policyContentStruct trust.PolicyContent, systemRegistri
 				Type:      trustTypeDescription(repoval[0].Type),
 			}
 			// TODO - keyarr is not used and I don't know its intent; commenting out for now for someone to fix later
```

Comment:
This is two years old (May 2020). I think if noone has used `keyarr` by now it can just be removed.

Reply:
I would prefer to not do this with this PR.